src/main/user-impl/java/com/mysql/cj/jdbc/AbandonedConnectionCleanupThread.java

```diff
@@ -39,17 +45,16 @@
 import java.util.concurrent.locks.Lock;
 import java.util.concurrent.locks.ReentrantLock;
 
-import com.mysql.cj.MysqlConnection;
-import com.mysql.cj.conf.PropertyDefinitions;
-import com.mysql.cj.protocol.NetworkResources;
-
 /**
  * This class implements a thread that is responsible for closing abandoned MySQL connections, i.e., connections that are not explicitly closed.
  * There is only one instance of this class and there is a single thread to do this task. This thread's executor is statically referenced in this same class.
  */
 public class AbandonedConnectionCleanupThread implements Runnable {
     private static final Set<ConnectionFinalizerPhantomReference> connectionFinalizerPhantomRefs = ConcurrentHashMap.newKeySet();
-    private static final ReferenceQueue<MysqlConnection> referenceQueue = new ReferenceQueue<>();
+    private static final ReferenceQueue<MysqlConnection> connectionReferenceQueue = new ReferenceQueue<>();
```

Comment:
I don't recommend adding these changes until we have tested it to know it does what we want it to do. I recommend putting this in a separate PR so we can merge the other changes in first.

Reply:
Removed from class.